### ISSUE TITLE
latest docker image from paddlepaddle/paddle:latest can't import fluid

### ISSUE THREAD

**Issue by jetfuel** [PaddlePaddle/Paddle#11059]
I am trying to run the paddle docker image to test paddle fluid. 
So first I pull the docker images with 
```
docker pull paddlepaddle/paddle
``` 

Once the update is completed. I tried to run  `paddlepaddle/paddle:latest` and import `paddle.fluid` and got the following error. 

```
(snake2.7) C02QF201G8WM-JW:book wangjeff$ docker run -it -p 8888:8888 -v $PWD:/work paddlepaddle/paddle:latest /bin/bash
root@947305d4983d:/# python
Python 2.7.12 (default, Dec  4 2017, 14:50:18) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import paddle
>>> import paddle.v2
>>> import paddle.fluid
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/paddle/fluid/__init__.py", line 17, in <module>
    import framework
  File "/usr/local/lib/python2.7/dist-packages/paddle/fluid/framework.py", line 22, in <module>
    from . import core
ImportError: /usr/local/lib/python2.7/dist-packages/paddle/fluid/core.so: undefined symbol: _Z32TouchOpRegistrar_tensorrt_enginev
>>> 
```

Looks like the linking is missing the tensorrt_engine_op.h. Since tensorrt_engine_op.h is a pretty new file. I suspect that we just miss the configuration for it.

**Comment by Superjomn** [comment]
Is this compiled on mac?  Our CI should use python unit test and run such cases.

**Comment by Superjomn** [comment]
It's weird, the CI system should have tested such cases with python unit tests. 
I found something wrong with the CMakeList.txt and will open a PR latter.

**Comment by luotao1** [comment]
fixed by #11050
